docs/modules/user-guide/partials/proc_devfile-registry.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc_devfile-registry_{context}"]
+= Devfile registry
+
+[role="_abstract"]
+A devfile registry is a service that stores and provides devfile stacks to Kubernetes developer tools like odo, Eclipse Che and the OpenShift Developer Console. Therefore, users can access devfiles through the devfile registry.  Devfile registries are based on the Oracle Cloud Infrastructure (OCI) Specification, and devfile stacks are stored as OCI-artifacts in the registry.
+
+Each devfile stack corresponds to a specific runtime or framework, such as,  NodeJS, Quarkus, and Wildfly. A devfile stack also contains resources, such as, a devfile.yaml, logo, and outerloop resources. These devfile stacks give users a template so they can get started developing with devfiles. For more information on the structure of a devfile registry, see https://github.com/johnmcollier/registry-docs/blob/main/registry-structure.adoc[registry-structure].
+
+== Devfile registry components
+
+A devfile registry is composed of two components, a devfile index server and an OCI registry server.
+
+.Devfile index server
+
+The server, and the resulting container image, does the following:
```

Comment:
IBM SG: "**following** adjective, noun
To introduce a list, procedure, or subsequent items, always use "following" as an adjective before a noun. For example, do not write "Complete the following" or "The functions include the following". Instead, write
"Complete the following steps" or "The adapter has the following functions"."

Reply:
Thanks, Rolfe, I've made the change, let me know what you think, thanks!